data/json/recipes/recipe_medsandchemicals.json

```diff
@@ -1658,5 +1658,50 @@
     "tools": [ [ [ "syringe", -1 ] ], [ [ "surface_heat", 30, "LIST" ] ] ],
     "components": [ [ [ "venom_paralytic", 1 ] ] ],
     "book_learn": [ [ "textbook_toxicology", 5 ], [ "textbook_botany", 5 ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "chem_turpentine",
+    "byproducts": [ [ "rosin", 1 ] ],
```

Comment:
Just want to be sure that it's not a typo. Did you mean resin?

Reply:
Its meant to be rosin. Turpentine is the volatile oils, rosin is the goop left behind. The seive is due to how there'll be a bunch of splinters and crap with how the pine resin is harvested which need to be seperated before allowing the rosin to harden.